src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/ClinicalVariableCard.js

```diff
@@ -0,0 +1,533 @@
+              ? 'Not enough data'
+              : selectedSurvivalBins.indexOf(bin.key) > -1
+                ? `Click icon to remove "${bin.groupName || bin.key}"`
+                : selectedSurvivalBins.length < MAXIMUM_CURVES
+                  ? `Click icon to plot "${bin.groupName || bin.key}"`
+                  : `Maximum plots (${MAXIMUM_CURVES}) reached`
+          }
+          >
+          <Button
+            disabled={
+              bin.key === '_missing' ||
+              bin.chart_doc_count < MINIMUM_CASES ||
+              (selectedSurvivalBins.length >= MAXIMUM_CURVES &&
+                selectedSurvivalBins.indexOf(bin.key) === -1)
+            }
+            onClick={() => {
+              updateSelectedSurvivalBins(displayData, bin);
+            }}
+            style={{
+              backgroundColor:
+                selectedSurvivalBins.indexOf(bin.key) === -1
+                  ? '#666'
+                  : colors(selectedSurvivalBins.indexOf(bin.key)),
+              color: 'white',
+              margin: '0 auto',
+              opacity:
+                bin.key === '_missing' ||
+                  bin.chart_doc_count < MINIMUM_CASES ||
+                  (selectedSurvivalBins.length >= MAXIMUM_CURVES &&
+                    selectedSurvivalBins.indexOf(bin.key) === -1)
+                  ? '0.33'
+                  : '1',
+              padding: '2px 3px',
+              position: 'static',
+            }}
+            >
+            {selectedSurvivalLoadingIds.indexOf(bin.key) !== -1
+              ? <SpinnerIcon />
+              : <SurvivalIcon />}
+            <Hidden>add to survival plot</Hidden>
+          </Button>
+        </Tooltip>
+      ),
+    },
+  ))
+);
+
+const getBoxTableData = (data = {}) => (
+  Object.keys(data).length
+    ? sortBy(Object.keys(data), datum => boxTableAllowedStats
+      .indexOf(datum.toLowerCase()))
+      .reduce(
+        (tableData, stat) => (
+          boxTableAllowedStats.includes(stat.toLowerCase())
+            ? tableData.concat({
+              count: parseContinuousValue(data[stat]),
+              stat: boxTableRenamedStats[stat] || stat, // Shows the descriptive label
+            })
+            : tableData
+        ), []
+      )
+    : []
+);
+
+const ClinicalVariableCard = ({
+  boxPlotValues,
+  currentAnalysis,
+  dataBuckets,
+  dataDimension,
+  dispatch,
+  dispatchUpdateClinicalVariable,
+  displayData,
+  fieldName,
+  filters,
+  getContinuousBins,
+  id,
+  openCustomBinModal,
+  overallSurvivalData,
+  plots,
+  qqData,
+  resetBins,
+  binsAreCustom,
+  selectedBins,
+  selectedSurvivalBins,
+  selectedSurvivalData,
+  selectedSurvivalLoadingIds,
+  setId,
+  setQQData,
+  setQQDataIsSet,
+  setSelectedBins,
+  style = {},
+  survivalPlotLoading,
+  theme,
+  totalDocs,
+  updateSelectedSurvivalBins,
+  variable,
+  wrapperId,
+}) => {
+  const tableData = variable.active_chart === 'box'
```

Comment:
at the moment - no, because `boxPlotValues` and `tableData` need access to the card's local state

this would be a good improvement once I have the selected survival values in redux